docs/UserGuide.md

```diff
@@ -123,3 +123,11 @@ Format: `genre INDEX c/ACTION g/GENRE [g/GENRE]`
   Examples:
 *  `genre 1 c/add g/shounen g/medieval fantasy g/isekai`
 *  `genre 1 c/delete g/shounen g/medieval fantasy g/isekai`
+
+### Clearing all animes currently displayed: `clear`
+
+Displays a confirmation prompt to user asking whether they wish to proceed with clearing the displayed animes.<br>
+User can choose to proceed by entering 'clear' again,
```

Comment:
Note entering "clear " (with whitespaces will work fine), but so will other commands, so I'm not sure if we should note that down.